application/whatsapp/v0/config/definition.json

```diff
@@ -0,0 +1,27 @@
+{
+  "availableTasks": [
+    "TASK_SEND_TEXT_BASED_TEMPLATE_MESSAGE",
+    "TASK_SEND_MEDIA_BASED_TEMPLATE_MESSAGE",
+    "TASK_SEND_LOCATION_BASED_TEMPLATE_MESSAGE",
+    "TASK_SEND_AUTHENTICATION_TEMPLATE_MESSAGE",
+    "TASK_SEND_TEXT_MESSAGE",
+    "TASK_SEND_MEDIA_MESSAGE",
+    "TASK_SEND_LOCATION_MESSAGE",
+    "TASK_SEND_CONTACT_MESSAGE",
+    "TASK_SEND_INTERACTIVE_CALL_TO_ACTION_URL_BUTTON_MESSAGE"
```

Comment:
- We do not have read message because the only way to "read" message from clients is to actually receive those messages through the webhook endpoints we have set up. To sum it up, it is not possible to extract messages from a chatroom, and that we can only read when we receive it through webhook endpoints.
- Regarding group chatroom, it is not possible to add the phone numbers registered in WhatsApp Business API to a group.